src/components/home/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import Home from './home'
+
+export default class Home extends React.Component {
+  render() {
+    return <Home />
+  }
+}
```

Comment:
You are complicating things.
Remove react import.
and then just export home.
```
import Home from './home'

export default Home
```
That's it.